app/controllers/application_controller.rb

```diff
@@ -11,4 +13,17 @@ def check_user
   def admin?
     redirect_to root_path, alert: "Not Authorized" unless current_user.admin
   end
+
+  def side_bar_posts
+    if home_page?
+      @sidebar_posts = Post.last_ten.drop(3)
+    else
+      @sidebar_posts = Post.last_ten.take(7)
+    end
+  end
+
+  private
+  def home_page?
+    params[:controller] == 'home' && params[:action] == 'index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.